server/src/services/item.service.ts

```diff
@@ -8,6 +11,38 @@ class ItemService {
       },
     });
   }
+
+  async getAll(sortType: SortType): Promise<Item[]> {
+    return Item.findAll({
+      attributes: {
+        exclude: ['images', 'detail', 'views', 'created_at', 'updated_at'],
+      },
```

Comment:
아래 `getFiltered()` 메소드와 동일한 return value라서 `include`와 `exclude` 중 하나로 통일하는 게 어떨까요??


```suggestion
      attributes: ['id', 'title', 'price', 'original_price', 'thumbnail', 'tags'],
```

Reply:
두가지가 크게 차이가 나지 않는 경우에 한해서 한쪽으로 통일하는 것이 확실히 좋겠군요!
기본적으로 include로 작성하되 차이가 심할 경우에만 exclude로 하는 편이 좋으려나요? 🤔
우선 제안주신 코드를 반영하겠습니당 ㅎㅎ